src/View/batchprocessing/ROINameCleaningOptions.py

```diff
@@ -175,46 +171,87 @@ def get_standard_names(self):
+        self.table_organ.horizontalHeaderItem(2).setTextAlignment(
             QtCore.Qt.AlignLeft)
-        self.table_roi.horizontalHeaderItem(3).setTextAlignment(
+        self.table_organ.horizontalHeaderItem(3).setTextAlignment(
             QtCore.Qt.AlignLeft)
 
         # Set header stretch
-        roi_name_header = self.table_roi.horizontalHeader()
+        roi_name_header = self.table_organ.horizontalHeader()
         roi_name_header.setSectionResizeMode(0, QtWidgets.QHeaderView.Stretch)
         roi_name_header.setSectionResizeMode(1, QtWidgets.QHeaderView.Stretch)
         roi_name_header.setSectionResizeMode(2, QtWidgets.QHeaderView.Stretch)
         roi_name_header.setSectionResizeMode(3, QtWidgets.QHeaderView.Stretch)
 
         # Removing the ability to edit tables with immediate click
-        self.table_roi.setEditTriggers(
+        self.table_organ.setEditTriggers(
             QtWidgets.QTreeView.NoEditTriggers |
             QtWidgets.QTreeView.NoEditTriggers)
 
         # Add table to the main layout
-        self.main_layout.addWidget(self.table_roi)
+        self.main_layout.addWidget(self.table_organ)
+
+    def create_table_view_volume(self):
+        """
+        Create a table to display all of the non-standard ROIs and
+        options for what to do with them.
+        """
+        # Create table
+        self.table_volume = QtWidgets.QTableWidget(self)
+        self.table_volume.setStyleSheet(
+            "background-color: rgb(255, 255, 255);")
+        self.table_volume.setColumnCount(4)
+        self.table_volume.verticalHeader().hide()
+        # Note - "New Name" is only enabled if the option "Rename" is
+        # selected.
+        self.table_volume.setHorizontalHeaderLabels(
+            [" Volume Prefix ", " Option ", " New Name ", " Dataset Location "])
+
+        # Set text align
+        self.table_volume.horizontalHeaderItem(0).setTextAlignment(
+            QtCore.Qt.AlignLeft)
+        self.table_volume.horizontalHeaderItem(1).setTextAlignment(
+            QtCore.Qt.AlignLeft)
+        self.table_volume.horizontalHeaderItem(2).setTextAlignment(
+            QtCore.Qt.AlignLeft)
+        self.table_volume.horizontalHeaderItem(3).setTextAlignment(
+            QtCore.Qt.AlignLeft)
+
+        # Set header stretch
+        roi_name_header = self.table_volume.horizontalHeader()
+        roi_name_header.setSectionResizeMode(0, QtWidgets.QHeaderView.Stretch)
+        roi_name_header.setSectionResizeMode(1, QtWidgets.QHeaderView.Stretch)
+        roi_name_header.setSectionResizeMode(2, QtWidgets.QHeaderView.Stretch)
+        roi_name_header.setSectionResizeMode(3, QtWidgets.QHeaderView.Stretch)
+
+        # Removing the ability to edit tables with immediate click
+        self.table_volume.setEditTriggers(
+            QtWidgets.QTreeView.NoEditTriggers |
+            QtWidgets.QTreeView.NoEditTriggers)
+
+        # Add table to the main layout
+        self.main_layout.addWidget(self.table_volume)
```

Comment:
needs an info log here to specify the function has successfully completed

Reply:
Fixed